test/test_scytl/test_scytl.py

```diff
@@ -0,0 +1,253 @@
+import unittest
+import os
+import requests_mock
+import csv
+from parsons.scytl import Scytl, scytl
+
+TEST_STATE = 'GA'
+TEST_ELECTION_ID = '114729'
+TEST_VERSION_NUM = '296262'
+
+_dir = os.path.dirname(__file__)
+
+
+class TestScytl(unittest.TestCase):
+
+    @requests_mock.Mocker()
+    def test_get_summary_results_succeeds(self, m):
+        self._mock_responses(m)
+
+        mock_url = scytl.summary_csv_url_template.format(
+            administrator=TEST_STATE, election_id=TEST_ELECTION_ID, version_num=TEST_VERSION_NUM
+        )
+
+        with open(f'{_dir}/114729_summary.zip', 'r') as summary:
+            m.get(mock_url, body=summary)
+
+        scy = Scytl(TEST_STATE, TEST_ELECTION_ID)
+        result = scy.get_summary_results()
+
+        with open(f'{_dir}/114729_summary_expected.csv', 'r') as expected:
+            expectedResult = list(csv.DictReader(expected, delimiter=",", ))
+
+            for i, row in enumerate(result):
+                expectedResultRow = expectedResult[i]
+
+                expectedResultRow['counties_reporting'] = \
+                    expectedResultRow['counties_reporting'] or None
+                expectedResultRow['total_counties'] = expectedResultRow['total_counties'] or None
+
+                self.assertDictEqual(row, expectedResultRow)
+
+    @requests_mock.Mocker()
+    def test_get_summary_results_skips_if_no_version_update(self, m: requests_mock.Mocker):
+        self._mock_responses(m)
+
+        scy = Scytl(TEST_STATE, TEST_ELECTION_ID)
+
+        result = scy.get_summary_results()
+
+        assert result is not None
```

Comment:
prefer unittest asserts over the `assert` operation since they give more specific error messages.